releasenotes/notes/add-5b9f14b11d8992b9.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Adds new keyword argument "mirror" to ``SUCCD`` class.
+    .. code-block:: python
+    
+      from qiskit_nature.circuit.library.ansatzes import SUCCfull
+      from qiskit_nature.converters.second_quantization import QubitConverter
+      from qiskit_nature.mappers.second_quantization import JordanWignerMapper

```

Comment:
Please correct these imports

Reply:
I also just noticed that the name of this file is rather non-descriptive. I'm not sure what `reno` is doing with those hashes at the end of the file names.. @manoelmarques is it safe to rename this file?